packages/jsii/lib/assembler.ts

```diff
@@ -479,11 +505,47 @@ export class Assembler implements Emitter {
         return _sortMembers(this._visitDocumentation(type.symbol, jsiiType));
     }
 
-    private async _visitEnum(type: ts.Type, namespace: string[]): Promise<spec.EnumType> {
+    /**
+     * @returns true if this member is internal and should be omitted from the type manifest
+     */
+    private _isPrivateOrInternal(symbol: ts.Symbol, validateDeclaration?: ts.Declaration): boolean {
+        const hasInternalJsDocTag = _hasInternalJsDocTag(symbol);
+        const hasUnderscorePrefix = symbol.name !== '__constructor' && symbol.name.startsWith('_');
+
+        if (_isPrivate(symbol)) {
+            LOG.trace(`skipping ${symbol.name} because it is marked "private"`);
```

Comment:
That trace here might be misleading - this is just returning a boolean, you cannot presume what is done with it.

Reply:
Fixed.